src/pages/templates/pageSections/hero/imageWithTextAndCTA.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Button, Image, Text, VStack, useBreakpointValue} from "@chakra-ui/react";
+
+export default function ImageWithTextAndCTA(){
+
+  const height = useBreakpointValue({ base: "250px", md: "400px" })
+  const btnSize = useBreakpointValue({ base: "md", md: "lg" })
+  const fontSize = useBreakpointValue({ base: "2xl", md: "4xl" })
+  const pl = useBreakpointValue({ base: "1rem", md: "3.5rem" })
+
+  return (
+    <Box w={"100vw"} pos={"relative"} h={height}>
+      <Image 
+        src={"https://placeimg.com/1000/480/tech/grayscale"} 
+        w={"100%"}
+        h={"100%"} 
+        objectFit={"cover"}/>
+      <VStack 
+        align={"start"} 
+        pos={"absolute"} 
+        top={"0"} 
+        h={"100%"} 
```

Comment:
I would suggest to also add a `w={"full"}` so the gradient is covering the whole image.

Reply:
I actually wanted the gradient to just cover the text and not the whole image. 
Do you think it'll be better to make it the other way?